aproximation_rules.py

```diff
@@ -0,0 +1,58 @@
+import numpy

```

Comment:
people that use numpy like importing it like this:
import numpy as np, it's just a convention at this point.